users/muppetjones/rules.mk

```
@@ -0,0 +1,17 @@
+# Reduce firmware size
+# 	https://thomasbaart.nl/2018/12/01/reducing-firmware-size-in-qmk/
+# also requires in config.h
+#	NO_ACTION_MACRO
+#	NO_ACTION_FUNCTION
+EXTRAFLAGS += -flto
```

You want to be using the "official" way, here and elsewhere, instead
```suggestion
LTO_ENABLE = yes
```